### DISCUSSION TITLE
It would be great if you could add line(curve) control in Custom Curve Scaffold.

### DISCUSSION THREAD

**Discussion by MusicDendy** [zBritva/charticulator-visual-community#14] [Ideas]
![image](https://github.com/zBritva/charticulator-visual-community/assets/24212122/7de1d6b7-c841-40e5-bfd4-8f9d2c183983)
1.Now we can not see and change(fix) the curve in Custom Curve Scaffold.
2. Can you implement control of this curve and the possibility of simplifying this curve, for example as we have in Word?
![image](https://github.com/zBritva/charticulator-visual-community/assets/24212122/fe9e77e3-4ea9-4c8f-8503-14c8f9d3902a)
3.For example now it is very difficult to draw simple line.

**Comment by zBritva** [comment]
It sounds good
